packages/chat-component/src/components/chat-component.ts

```diff
@@ -108,6 +119,26 @@ export class ChatComponent extends LitElement {
 
   static override styles = [chatStyle];
 
+  override updated(changedProperties: Map<string | number | symbol, unknown>) {
```

Comment:
More food for thoughts: maybe we could skip that part entirely and allow to directly override the CSS variables through CSS?

Reply:
Let's think about it. We can have a follow up to this.

Reply:
I opened a follow up issue, btw.

Reply:
>> https://github.com/Azure-Samples/azure-search-openai-javascript/pull/160#discussion_r1422744169

I kind of disagree here, though, because it's a 'isCustomBrandingEnabled' as 'isDarkThemeEnabled'. We can assume that the app 'hasCustomBranding', wether it's on or off, regardless. But this indicates it is enabled when true.